Environment variable saving note (#353)
astronomer/docs: cloud/environment-variables.md

3. Specify an environment variable key and value in the table. For sensitive credentials that should be treated with an additional layer of security, select the **Secret** checkbox. This will permanently hide the variable's value from all users in your Workspace.

    When you're finished configuring the environment variable, click **Add**.

    ![Add Variables button highlighted in the Environment Variables configuration table](/img/docs/add-variable.png)

> 4. Click **Update Variables** to save your changes. This action restarts your Airflow Scheduler, Webserver, and Workers. After saving, it can take up to two minutes before the new variables are live in your Deployment.

Reviewer: ```suggestion
4. Click **Update Variables** to save your changes. This action restarts your Airflow Scheduler and Workers. After saving, it can take up to two minutes for the new variables to be available and applied to your Deployment.
```

Author: Trying to keep the voice active here and more simple, is there enough of a difference between "available" and "applied" that we need to mention both?

    ![Green checkmark icon next to an updated value in the Environment Variables configuration table](/img/docs/variable-checkmark.png)

> 5. Click **Update Variables** to save your changes. This action restarts your Airflow Scheduler, Webserver, and Workers. After saving, it can take up to two minutes before the updated variables are live in your Deployment.

Reviewer: ```suggestion
5. Click **Update Variables** to save your changes. This action restarts your Airflow Scheduler and Workers. After saving, it can take up to two minutes for the new variables to be available and applied to your Deployment.
```
